rules-tests/Restoration/Rector/ClassLike/UpdateFileNameByClassNameFileSystemRector/UpdateFileNameByClassNameFileSystemRectorUsingMultipleFilesChangedTraitTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\Restoration\Rector\ClassLike\UpdateFileNameByClassNameFileSystemRector;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class UpdateFileNameByClassNameFileSystemRectorUsingMultipleFilesChangedTraitTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $smartFileInfo): void
+    {
+        $this->doTestFileInfoWithAdditionalChanges($smartFileInfo);
+    }
+
+    /**
+     * @return Iterator<SmartFileInfo>
+     */
+    public function provideData(): Iterator
+    {
+        return $this->yieldFilesFromDirectory(__DIR__ . '/FixtureUsingMultipleFilesChangedTrait');
+    }
+
+    public function provideConfigFilePath(): string
+    {
+        return __DIR__ . '/config/configured_rule.php';
```

Comment:
I can't find this config in the PR. Where is it?

Reply:
I see, I used the same from `UpdateFileNameByClassNameFileSystemRectorTest` to approve the result is the same

Reply:
Could you separate it, so it's independent on each other? Now it might work, but each might break for 2 different reasons.

Reply:
done